docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -2685,3 +2704,407 @@ At this point, the circuit (Alice and Bob's gameroom) is ready. Next, the Acme
+(see [section I-2.3](#i-23-gameroom-rest-api-sends-a-circuitmanagementpayload)),
+the Gameroom daemon that creates the circuit definition (in this case, Acme's
+`gameroomd`) specifies its own public key as the scabbard admin. Since the Acme
+Gameroom daemon has the only key that’s authorized to add smart contracts, it
+is responsible for adding the XO smart contract.
+
+To add the XO smart contract, the Acme Gameroom daemon creates a series of
+transactions to set the permissions surrounding the smart contract and to
+submit the smart contract itself. For more information, see the
+[Sawtooth Sabre](https://sawtooth.hyperledger.org/docs/sabre/nightly/master/sabre_transaction_family.html)
+documentation.
+
+The Acme Gameroom daemon bundles these transactions into a batch, serializes
+the batch, and submits the serialized batch to the scabbard service on its
+local Splinter node:
+
+```
+	POST /scabbard/<circuit_id>/<service_id>/batches
+  <serialized batch>
+```
+
+For more information about batches, see "Transactions and Batches" in the
+[Sawtooth Architecture](https://sawtooth.hyperledger.org/docs/core/releases/latest/architecture/transactions_and_batches.html)
+documentation.
+
+When the Acme scabbard service receives this batch, it must agree with the
+Bubba Bakery scabbard service to submit the smart contract. The Acme scabbard
+service performs the following steps:
+
+a. Deserializes the batch: Shares the batch with the other scabbard services in
+   the circuit (in this case, the Bubba Bakery scabbard service)
+
+b. Creates a consensus proposal to send to the admin services of the other
+   nodes (in this case, Bubba Bakery's scabbard service) to agree on the batch
+
+c. Waits for consensus to agree on the batch, then commits it to scabbard
+   state. For information on consensus, see Appendix B: Consensus.
+
+d. After the scabbard services on both nodes have committed the smart contract,
+   the Acme Splinter node is done setting up the gameroom.
+
+#### Ⅰ-6.7. Both Gameroom daemons update gameroom status in database
+
+The application authorization handler listens for scabbard events using its
+state delta processor, `XoStateDeltaProcessor`. When the state delta processor
+receives an event with the address of the uploaded XO contract it then sets the
+status of the gameroom to “circuit_active”
+
+```
+StateChangeEvent containing contact address
+{
+  “type”: “Set”,
+  “message”: {
+    “key”: “<xo_contract_address>”
+    “value” [..]
+  }
+}
+```
+
+At this point, the state delta processor will begin listening for
+XO game state changes (defined in Appendix D).
+
+#### Ⅰ-6.8. Acme Gameroom daemon notifies Acme UI
+
+After the state delta processor sets the status of the gameroom to
+“circuit_active”, the Acme Gameroom application authorization handler adds a
+new entry to the `gameroom_notification` table:
+
+
+<table class ="gameroom_db_table" border="1">
+  <tr class="gameroom_db_headers">
+    <th><code>id</code></th>
```

Comment:
The tables need to be reworked. In several instances (this one included), what were multiple tables in the PDF show up as a signal weird table in markdown.

The headers should also not be marked <code>. Its not clear if the data content should be marked code either; it may look better visually without.

Reply:
Updated to separate the bigger tables into separate tables, and also updated all of the tables to remove that code formatting.